src/sezzle-checkout-button/sezzle-checkout-button-asset.js

```diff
@@ -3,6 +3,10 @@ class SezzleCheckoutButton {
 	constructor(options){
 		this.theme =  options.theme  || 'light';
 		this.template = options.template || 'Checkout with %%logo%%';
+		this.eventLogger = new EventLogger({
+			merchantUUID: options.merchantUUID,
+			widgetServerEventLogEndPoint: options.widgetServerEventLogEndPoint
```

Comment:
Endpoint is one word, so 'point' can be lowercase

Reply:
addressed